test/sanity/validate-modules/validate-modules

```diff
@@ -41,6 +41,14 @@ from schema import doc_schema, option_schema, metadata_schema
 from utils import CaptureStd, parse_yaml
 from voluptuous.humanize import humanize_error
 
+from ansible.compat.six import PY3, with_metaclass
+
+if PY3:
+    # Because there is no ast.TryExcept in Python 3 ast module
+    ast.TryExcept = ast.Try
```

Comment:
I think I would prefer instead of faking this attribute, we create a var like:

```
if PY3:
    TRY_EXCEPT = ast.Try
else:
    TRY_EXCEPT = ast.TryExcept
```

And then use `TRY_EXCEPT` in the code.